ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/claimantresponse/CourtDeterminationMapper.java

```diff
@@ -37,10 +37,8 @@ public CourtDetermination from(CCDCourtDetermination ccdCourtDetermination) {
             .courtDecision(paymentIntentionMapper.from(ccdCourtDetermination.getCourtDecision()))
             .disposableIncome(ccdCourtDetermination.getDisposableIncome())
             .courtPaymentIntention(paymentIntentionMapper.from(ccdCourtDetermination.getCourtIntention()))
-            .decisionType(ccdCourtDetermination.getDecisionType());
-        if (ccdCourtDetermination.getRejectionReason() != null) {
-            builder.rejectionReason(ccdCourtDetermination.getRejectionReason());
-        }
+            .decisionType(ccdCourtDetermination.getDecisionType())
+            .rejectionReason(ccdCourtDetermination.getRejectionReason());
```

Comment:
rejection reason doesn't exist in most cases, that's the reason we were doing null check before i guess. Is it ok to remove it?

Reply:
Kiran it would be set null in this case which is also by default initialised as null, so it doesn't changes anything, good point of using builder and Optional in domain

Reply:
thanks for explaining guys 👍

Reply:
You are welcome. 👍